helper/config-value

```diff
@@ -0,0 +1,11 @@
+# needed_by: git-fork
+
+get_config_value() {
+  echo $(git config git-extras.$1)
+}
+
+# set_config_value "key" "value" "global|system" (global or system not required)
+set_config_value() {
+  $(git config ${3+--$3} git-extras.$1 $2)
+  echo $?
```

Comment:
IMHO, I prefer to use `git config ${3+--$3} git-extras.$1 $2` directly.
When writing shell script, people are more relied on  `$?` that the return value to check if the previous command succeeded.